crates/edr_evm/src/spec.rs

```diff
@@ -276,6 +279,9 @@ pub trait RuntimeSpec:
     /// if it is associated with this chain specification.
     fn chain_hardfork_activations(chain_id: u64) -> Option<&'static Activations<Self::Hardfork>>;
 
+    /// Returns the base fee params corresponding to the provided chain ID
+    fn chain_base_fee_params(chain_id: u64) -> &'static BaseFeeParams<Self::Hardfork>;
+
     /// Returns the name corresponding to the provided chain ID, if it is
     /// associated with this chain specification.
     fn chain_name(chain_id: u64) -> Option<&'static str>;
```

Comment:
Question: Can we simplify the API by returning the `ChainConfig` instead of a single function per member of the `ChainConfig`?